docs/source/workflow_inputs.rst

```diff
@@ -117,7 +117,7 @@ These features may require shorter timesteps, allow more sophisticated simulatio
          Values greater than 1.0 have the effect of smoothing or damping the rate of change in the indoor air temperature from timestep to timestep.
          This heat capacitance effect is modeled on top of any other individual mass inputs (e.g., furniture mass, partition wall mass, interior drywall, etc.) in the HPXML.
   .. [#] DefrostModelType choices are "standard" and "advanced".
-  .. [#] Use "standard" for default E+ defrost setting.
+  .. [#] Use "standard" for defrost model per RESNET MINHERS Addendum 82, with integrated capacity and power approach.
```

Comment:
Not sure if this is the best place to put the descriptions considering we're likely to remove the DefrostModelType input.